src/main/flight/position.c

```diff
@@ -86,6 +90,8 @@ PG_RESET_TEMPLATE(positionConfig_t, positionConfig,
     .altitude_prefer_baro = 100, // percentage 'trust' of baro data
     .altitude_lpf = 300,
     .altitude_d_lpf = 100,
+    .hover_throttle = 1275,
+    .landing_altitude_m = 4,
```

Comment:
This needs version increase in `PG_REGISTER_WITH_RESET_TEMPLATE(positionConfig_t, positionConfig, PG_POSITION, 4);`

Reply:
Actually to 5 I think.

Reply:
Yes - was only showing the line (without the change).